src/ros_license_toolkit/main.py

```diff
@@ -98,16 +98,9 @@ def main(args: Optional[Sequence[str]] = None) -> int:
         results_per_package.update(
             process_one_pkg(rll_print, package))
 
-    # Generate copyright file
-    if parsed_args.generate_copyright_file:
-        if len(packages) == 1:
-            package = packages[0]
-            package.write_copyright_file(
-                os.path.join(os.getcwd(), 'copyright'))
-        else:
-            rll_print(red(
-                "Can only generate copyright file for single package"),
-                Verbosity.QUIET)
+    if max(results_per_package.values()) != Status.FAILURE:
+        if parsed_args.generate_copyright_file:
+            generate_copyright_file(packages, rll_print)
```

Comment:
```suggestion
        if parsed_args.generate_copyright_file:
            generate_copyright_file(packages, rll_print)
        else:
            rll_print(red(
                "Copyright file will not be generated because there were linter errors."),
            Verbosity.QUIET)
```

Reply:
or something like that ..